terraform/modules/database/database.tf

```diff
@@ -9,6 +9,13 @@ resource "aws_rds_cluster_instance" "cluster_instances" {
   auto_minor_version_upgrade = "true"
   performance_insights_enabled = "true"
   preferred_maintenance_window = "${var.preferred_maintenance_window}"
+  tags = "${merge(
+    var.default_tags,
+    map(
+      "Name","upload",
+      "Env","${var.deployment_stage}"
+    )
+  )}"
 }
 
```

Comment:
Avoiding the outer `"$(..)"`
```
tags = merge(
  var.default_tags,
  {
    "Name" = "upload",
    "Env"  = "${var.deployment_stage}"
  }
)
```

Reply:
It's throwing an error if i dont enclose it with "${}" :( It might be that we're using an older terraform version :( We might want a separate ticket to update the version.